demo/index.html

```diff
@@ -15,17 +15,17 @@
     <main></main>
 
     <script type="module">
-      import 'marked/marked.min.js';
-      import 'prismjs';
+      import 'https://unpkg.com/marked@latest/marked.min.js';
+      import 'https://unpkg.com/prismjs@latest/prism.js';
```

Comment:
You moved the code to use CDN resources, but the npm packages are still in the package.json?

Reply:
I have uninstalled the packages

Reply:
This should have been a change to continue to use the dependency version.

1. It's consistent with our other repos structure
2. Performance is faster the fewer http requests we have to make
3. Using latest on anything can result in future problems where an update to marked suddenly breaks our local host.
4. We are trying to move away from unpkg anyhow because of the reliability versions of it.